src/client/file_instance.cpp

```diff
@@ -106,6 +111,13 @@ void FileInstance::UnInitialize() {
 }
 
 int FileInstance::Read(char* buf, off_t offset, size_t length) {
+    if (!CheckIoAligned(offset, length, finfo_.blocksize)) {
```

Comment:
should use finfo_ in metacache， the fileinfo_ in FileInstance  may be incomplete.